src/node.h

```diff
@@ -471,6 +471,8 @@ struct IsolateSettings {
   v8::PromiseRejectCallback promise_reject_callback = nullptr;
   v8::AllowWasmCodeGenerationCallback
       allow_wasm_code_generation_callback = nullptr;
+  v8::ModifyCodeGenerationFromStringsCallback2
+      modify_code_generation_from_strings_callback = nullptr;
```

Comment:
This is currently semver-major because it changes the ABI, but you can make it semver-minor if you want by adding an opt-in flag to `IsolateSettingsFlags` above

Reply:
@addaleax should be set!